src/vector-integration.adoc

```diff
@@ -1,38 +1,27 @@
 [#section_vector_integration]
 == Integrating {cheri_base_ext_name} and {cheri_default_ext_name} with the Vector Extension
 
-The RISC-V vector (V) extension is generally orthogonal to CHERI because the
+The RISC-V vector (V) extension is orthogonal to CHERI because the
 vector registers only hold integer or floating-point data. The vector registers
-are not extended to hold capabilities. However, implementations must ensure
-that every memory access using vector instructions is authorized by a suitable
-capability, as described in xref:section_int_load_store_insns[xrefstyle=short],
-when integrating the vector extension with {cheri_base_ext_name} and
-{cheri_default_ext_name}.
+are _not_ extended to hold capabilities.
 
-Vector store instructions never write capabilities to memory. Also, the tag of
-any capabilities read from memory using vector load instructions is
-implicitly set to 0 as the data is written to a vector register regardless of
-the actual value in memory of that capability.
+NOTE: A future extension may allow tags to be stored in vector registers.
+  Until that time, vector load and store instructions must not be used to implement generic
+  memory copying in software, such as the `memcpy()` standard C library function,
+  because the vector registers do not hold capabilities, so the tags of any
+  copied capabilities will be set to 0 in the destination memory.
 
-NOTE: Vector load and store instructions must not be used to implement generic
-memory copying in software, such as the `memcpy()` standard C library function,
-because the vector registers do not hold capabilities, so the tags of any
-copied capabilities will be set to 0 in the destination memory.
+Vector loads and stores all following the behaviour as described in
+xref:section_int_load_store_insns[xrefstyle=short], as all are treated as integer data.
```

Comment:
Minor nitpicking: I avoided the expression "integer data" in the original version because technically vector registers can hold various formats of integer and floating-point data -- I recommend avoiding it here too!

Reply:
AFAIK vector loads don't do things like nan boxing on loaded FP data - I think they all load/store data as if it is integer data - so I think this is correct.... they certainly all follow the rules in that xref so I think this is ok as it is.

Reply:
also, if there's a move between V and X registers then it is always treated as integer data, which this sentence is also supposed to cover. Can you think of any counter-examples?

Reply:
I'm not sure the "as all are integer data" makes this sentence any clearer, could just drop it?

Reply:
```suggestion
xref:section_int_load_store_insns[xrefstyle=short].
```